capitoli/Quinto_Passo_Primi_Programmi.rst

```diff
@@ -0,0 +1,371 @@
+
+**Come facciamo a far in modo che l’interprete accetti qualunque carattere immesso dall'utilizzatore?**
+**Usiamo semplicemente il comando “input” senza specificare nulla.** 
+
+**Il programma seguente:** 
+
+::	
+
+      	s = input ("Come ti chiami? ")
+        print ("Ciao PAOLA", s)                       
+	 
+	#soluzione
+ 	       #Ciao Paola
+	       #Ciao Alda
+	       #Ciao Marco 
+
+Esercitati con gli esempi seguenti:
+
+::
+
+	print ("Alt! ")
+	s = input ("Chi va la'? ")
+	print ("Passa pure ", s)
+	num = int (input ("Scrivi un numero "))
+	print ("num = ", num)
+	print ("num * 2 = ", num * 2)
+
+        #soluzione
+
+**Esercitiamoci un po’** 
+
+Ci sono più soluzioni possibili per ognuno degli esercizi proposti; sta a te trovarle e, soprattutto, provarle. 
+
+1. Scrivi un programma che chiede un numero e ne calcola il quadrato e il cubo e li visualizza sullo schermo. 
+    
+2. Scrivi un programma che aggiunge 7 a qualunque numero inserito e visualizza il risultato sullo schermo. 
+    
+3. Scrivi un programma che chiede due numeri, li somma e visualizza il risultato. 
+   
+4. Scrivi il programma per calcolare l’area di qualunque rettangolo chiedendo all’utilizzatore la base e l’altezza. 
+   
+5. Scrivi il programma che chieda tre numeri e ne visualizzi sia la somma sia il prodotto. 
+   
+6. Scrivi il programma che calcola la metà e il doppio di qualunque numero inserito dall’utente, poi visualizza i risultati. 
+    
+7. Scrivi il programma che chiede la misura del lato di un quadrato e ne calcola l’area, poi visualizza il risultato. 
+   
+8. Scrivi il programma che calcola il perimetro del cortile della scuola che è un rettangolo i cui lati misurano rispettivamente 45 m e 65 m
+   visualizza il risultato. Quindi calcola il perimetro di ogni rettangolo per il quale l’operatore inserisca la misura della base e
+   dell’altezza. 
+   
+9. Scrivi un programma che chiede tre numeri, ne calcola la somma, la somma dei quadrati e il quadrato della somma. Infine, visualizza i
+   risultati. 
+
+.. activecode:: Esercizi
+   :coach:
+   :caption: Esercizi
+
+
+**ESERCIZI CON VALUTAZIONE**
+
+Concediamoci un momento di pausa per giocare un po’.
+Prima di proseguire il nostro percorso di studio, facciamo 
+un breve gioco. Giochiamo a:
+
+:boltred:`CACCIA ALL’ERRORE!`
+
+Regole del gioco: 
+In ogni programma è inserito un errore. 
+Leggi attentamente ciascun programma, prova a digitarlo utilizzando Python, scopri e correggi l’errore. 
+Per ogni esercizio assegnati un punto se riesci a trovare l’errore e un altro punto se riesci a correggerlo.
+
+------------------------------
+
+::
+
+       **Es. 1:** stampa il nome del tuo cantante preferito.
```

Comment:
Questo è un blocco di codice e quindi i ** non hanno effetto. Usa `#` per fare un commento quindi a riga 326 metti `# Es. 1 bla bla`. qui come in tutti gli altri casi successivi

Reply:
ma come faccio a non farla vedere nel testo @libremente

Reply:
Deve vedersi nel testo, è un commento che fa parte del codice Python che inserisci nel blocco.